bcgov/main.tf

```diff
@@ -58,7 +58,7 @@ resource "kubernetes_secret" "secret_sa" {
   count = length(google_storage_bucket.bucket)
   metadata {
     name      = "gcp-${google_storage_bucket.bucket[count.index].name}-service-account-key"
-    namespace = "${split(",", element(var.namespace_apps, count.index))[0]}"
+    namespace = split(",", element(var.namespace_apps, count.index))[0]
```

Comment:
If we're having to split this anyway, why not have two arguments? Not necessarily asking for a change but the thought process that led to this implementation.

Reply:
two separate arguments for
namespace_apps and namespaces or
namespaces and apps ?

Reply:
Perhaps I misunderstood this line?
> it requires one string argument includes namespace and app name separated by comma, `"namespace,app"` in `command` job attribute

I'm asking why not send `"namespace" "app"` as arguments to the job. Can we explain any design constraints that led us to go with `"namespace,app"`?

Reply:
I see your point, one of the terraform variable, namespace_apps, is an array of string separate by comma. so I used it as it is for the arguments but we can definitely allow two separate arguments for namespace and app and concatenate those two strings inside the script to add `namespace,app` value into `namespace_apps` variable.